cs/forms.texy

```diff
@@ -212,8 +205,8 @@ $form->addTextArea('note', 'Poznámka:')
 \--
 
 
-addUpload($name, $label = NULL) .{toc: addUpload()}
------------------------------------------------
+addUpload($name, $label = NULL, $multiple = FALSE) .{toc: addUpload()}
+----------------------------------------------------------------------
 
```

Comment:
Od verze 2.2.3 existuje přímo addMultiUpload, asi by bylo vhodnější zmínit přímo tento prvek a do poznámky přidat, že existuje až od této verze a předtím se dalo použít TRUE.